api/client/deals.go

```diff
@@ -70,22 +70,22 @@ func (d *Deals) Store(ctx context.Context, addr string, data io.Reader, dealConf
 		return nil, nil, err
 	}
 
-	cids := make([]cid.Cid, len(reply.GetCids()))
-	for i, replyCid := range reply.GetCids() {
+	cids := make([]cid.Cid, len(reply.GetProposalCids()))
```

Comment:
Renamed to `GetProposalCids()` since now there's a new attribute for the CID of the data stored, so to avoid confusions.

Reply:
Makes sense.